tools/scxtop/src/main.rs

```diff
@@ -62,6 +65,25 @@ struct Args {
     /// Process to monitor or all.
     #[arg(long, default_value_t = -1)]
     process_id: i32,
+
+    /// Automatically start a trace when a function takes too long to return.
+    #[arg(
+        long,
+        default_value_t = false,
+        requires("experimental_long_tail_tracing_symbol"),
+        requires("experimental_long_tail_tracing_binary")
+    )]
+    experimental_long_tail_tracing: bool,
+    /// Symbol to automatically trace the long tail of.
+    #[arg(long)]
+    experimental_long_tail_tracing_symbol: Option<String>,
+    /// Binary to attach the uprobe and uretprobe to.
+    #[arg(long)]
+    experimental_long_tail_tracing_binary: Option<String>,
+    /// Minimum latency to trigger a trace. Defaults to 0ns which launches a
+    /// trace on every return, this probably isn't what you want!
+    #[arg(long, default_value_t = 100000000)]
+    experimental_long_tail_tracing_min_latency_ns: u64,
```

Comment:
Should this error if the value is 0?

Reply:
Ah, forgot I changed the default, will update the comment. I think tracing every time a uprobe is hit is fine with the warning, even if the implementation that times it anyway is a bit silly.